src/plugins/intel_cpu/src/graph_optimizer.cpp

```diff
@@ -678,12 +682,63 @@ void GraphOptimizer::FuseMultiplyAndAdd(Graph& graph) {
     }
 }
 
-void GraphOptimizer::MergeConvertAndScaleShift(Graph& graph) {
+void GraphOptimizer::MergeEltwiseAndConvert(Graph& graph) {
+// The pass is required on arm platforms only
+#if !defined(OPENVINO_ARCH_ARM64)
+    return;
+#endif
+    auto& graphNodes = graph.GetNodes();
+
+    auto parent = graphNodes.begin();
+    while (parent != graphNodes.end()) {
+        CPU_GRAPH_OPTIMIZER_SCOPE(MergeEltwiseAndConvert);
+        auto parentNode = *parent;
+        if (parentNode->getType() != Type::Eltwise) {
+            parent++;
+            continue;
+        }
+
+        const auto& childEdges = parentNode->getChildEdges();
+        if (childEdges.size() != 1) {
+            parent++;
+            continue;
+        }
+
+        const auto edge = childEdges[0].lock();
+        auto childNode = edge->getChild();
+        if (childNode->getType() != Type::Convert) {
+            parent++;
+            continue;
+        }
+
+        if (!one_of(childNode->getOriginalOutputPrecisionAtPort(0),
```

Comment:
I just understood that we can fuse Convert into Eltwise only in case JIT impl is supported.
So lets create conFuseConvert method in Eltwise node. The method should call jitIsSupported fucntion.
Also lest move these precisions check into the newly added method.

Reply:
Fixed